tests/test-run.sh

```diff
@@ -80,6 +80,13 @@ assert_file_has_content runtime-fpi "^runtime=runtime/org\.test\.Platform/$ARCH/
 
 ok "run a runtime"
 
+run_sh org.test.Platform env | grep service_host_ >runtime-os-release
+while read -r line; do
+assert_file_has_content runtime-os-release "service_host_$(eval echo ${line})"
```

Comment:
```suggestion
assert_file_has_content runtime-os-release "service_host_$(eval "echo ${line}")"
```
?

Reply:
doesn't look like bash minds one way or the other :-) fixed

Reply:
(by which I mean the check was successful either way)